doc/modules/cross_validation.rst

```diff
@@ -209,6 +209,16 @@ The following sections list utilities to generate indices
 that can be used to generate dataset splits according to different cross
 validation strategies.
 
+Cross-validation iterators for IID data.
+========================================
+
+Data is Independent Identically Distributed (IID) if the individual samples
+of the data are independent of each other and are drawn from an identical
+distribution.
```

Comment:
Maybe something like "Simply put, it means each data point is created equally, and there are is no covariance between different data points." I'd also definitely add that this is standard scenario for most of machine learning.